packages/core/src/drivers/db/models/diligencia.js

```diff
@@ -0,0 +1,65 @@
+const { Model, DataTypes } = require('sequelize');
+const { USUARIO_TABLE } = require('./persona');
+
+const DILIGENCIA_TABLE = 'diligencia';
+
+const DiligenciaSchema = {
+  id: {
+    allowNull: false,
+    autoIncrement: true,
+    primaryKey: true,
+    type: DataTypes.INTEGER,
+  },
+  personaId: {
+    allowNull: false,
+    type: DataTypes.INTEGER,
+    field: 'persona_id',
+    references: {
+      model: USUARIO_TABLE,
+      key: 'id',
+    },
+  },
+
+  startTime: {
+    allowNull: false,
+    type: DataTypes.DATE,
+    field: 'start_time',
+  },
+
+  endTieme: {
+    allowNull: false,
+    type: DataTypes.DATE,
+    field: 'start_time',
+  },
+
+  updatedAt: {
+    type: DataTypes.DATE,
+    field: 'updated_at',
+    defaultValue: null,
+  },
+  deletedAt: {
+    type: DataTypes.DATE,
+    field: 'deleted_at',
+    defaultValue: null,
+  },
+};
+
```

Comment:
You forgot a field called createdAt, it is in all the others tables

Reply:
And please, add a field called, solicitudId because this will be associated with, many diligencias to one solicitud

Reply:
If it's a relationship from N:1 I think I shouldn't use the id here. In solicitudes model should have dilogencia_id